doc/compiler/jitserver/Build.md

```diff
@@ -128,6 +118,26 @@ bash get_source.sh -openj9-repo=https://github.com/<Your GitHub UserID>/openj9.g
 ```
 See https://www.eclipse.org/openj9/oj9_build.html for more detail. The only difference is you need to check out the JITServer code instead of upstream OpenJ9.
 
+You can test it by running the `java -version` command.
+
+```
+cd build/linux-x86_64-normal-server-release/images/jdk
+```
+Run:
+
+```
+./bin/java -version
+```
+Here is some sample output:
+
+```
+openjdk version "11.0.8-internal" 2020-07-14
```

Comment:
This is a bit confusing because above you are building a Java8 SDK (`git clone https://github.com/ibmruntimes/openj9-openjdk-jdk8.git`) but the version in here shows a Java11 build